### ISSUE TITLE
Audio stuttering when Google Map zoom in/out

### ISSUE THREAD

**Issue by iainheng** [gartnera/headunit#87]
Thanks for the recent bluetooth call fix. It works flawlessly except the video focus issue. However I am experiencing audio stuttering when navigating with Google Map particularly when zoom in and out. FYI, the issue doesn't occur in Waze in few of my tests. 

CMU version: 56
Android version: 8 Oreo (Google Pixel)
Car: Mazda 3 Skyactive 2014

**Comment by hansaya** [comment]
This happen to me as well while using waze but it seems random to me and I cannot reproduce consistently.

**Comment by mishan** [comment]
I wonder if it's running low on RAM or something. Did you open the stock Navigation app (it uses a ton of RAM)? I forget, there's a key combo to display memory usage and stuff on the CMU display -- or you could `ssh` in and run stuff like `w` and `free` to get an idea of how much utilization is happening.

These CMUs have some pretty underpowered hardware. I could be wrong, but it's worth ruling out.

**Comment by mishaaq** [comment]
Do you have a navigation card inserted? You could try removing it from the slot - it should be a far better. As mentioned by @mishan native navigation application consumes a lot of resources even when not opened.

**Comment by hansaya** [comment]
This started only after the recent update. Did any of the changes makes a different in ram consumption? I'm going to do more investigation this weekend

**Comment by Trevelopment** [comment]
is this a duplicate of #51?

**Comment by hansaya** [comment]
It seems like duplicate for me.
